Assets/LeapMotion/Modules/InteractionEngine/Scripts/UI/InteractionToggle.cs

```diff
@@ -131,5 +131,20 @@ public class InteractionToggle : InteractionButton {
     private void OnPressed() {
       isToggled = !isToggled;
     }
+
+    /// <summary>
+    /// Sets this InteractionToggle to the "toggled" state. Calling this function won't
+    /// oscillate the state of the toggle; to 'untoggle' the control, call Untoggle().
+    /// </summary>
+    public void Toggle() {
```

Comment:
Any reason for these to exist?  Doesn't `isToggled` already provide all of this functionality?

Reply:
Can't access properties in UnityEvents nor in tools like PlayMaker.

Reply:
As a practice, we should try to provide all API functionality as plain public methods for this reason. I don't see a way around it :\

Reply:
UnityEvents can definitely access properties, and as far as I'm aware, so can PlayMaker?  This page I found shows that PlayMaker can use a SetProperty method in order to modify the property of an object.

![image](https://user-images.githubusercontent.com/5723312/37310773-442ce510-2602-11e8-913b-267682a2dbce.png)

Reply:
OK well you straight up got me there with UnityEvents, I must have mixed it in with my PlayMaker experience with Martin.

I definitely remember Martin showing me an inability to manipulate properties when we were doing some IE work, but maybe there's just a feature we missed!

........But I still like having this in the API. Yes, it's duplicate functionality, but it's also an extremely direct and obvious usage for a "Toggle" object. I'd much rather have isToggled and Toggle()/Untoggle() both so that a user can just type "uhhh 'toggle.Toggle()', oh good, that's a function" and have it work.

Reply:
That makes sense, alright, I agree with it staying in!